pandas/core/arrays/datetimelike.py

```diff
@@ -224,8 +213,8 @@ class TimelikeOps:
 
             .. versionadded:: 0.24.0
 
-        nonexistent : 'shift_forward', 'shift_backward', 'NaT', timedelta, \
-default 'raise'
+        nonexistent : 'shift_forward', 'shift_backward', 'NaT', timedelta,
+            default 'raise'
```

Comment:
this is for proper rendering with numpydoc

Reply:
darn, will revert.